src/ruwaterloo_clj_scraper/core.clj

```diff
@@ -1,4 +1,5 @@
 (ns ruwaterloo-clj-scraper.core
+  (:refer-clojure :exclude [parse-opts])
```

Comment:
Are you sure this is needed?

Reply:
Yeah I get an "incorrect arity" error from cursive.

I read the solution from https://github.com/cursive-ide/cursive/issues/1678.